### PR TITLE
update to use latest dependencies

### PR DESCRIPTION
**Pull request by jFensch**
Updated to support Angular RC2

### CODE REVIEW

**Comment by MikeRyanDev** [comment]
Looks great! I'll test this over the weekend and get it merged and released.

**Comment by jFensch** [comment]
@MikeRyan52 where did you land with the PR?

File: package.json [hunk 1]
```diff
@@ -13,9 +13,10 @@
   "author": "Mike Ryan",
   "license": "MIT",
   "peerDependencies": {
-    "@ngrx/store": "^1.3.0",
-    "angular2": "^2.0.0-beta.8",
-    "rxjs": "^5.0.0-beta.2"
+    "@angular/core": "2.0.0-rc.2",
```
Comment (teropa) [inline package.json:16]: I'd use `^` here. Getting UNMET PEER DEPENDENCY on rc.4

**Comment by ricricucit** [comment]
Is this being maintained?...or is there any alternative solution to this problem?

**Comment by adrian-moisa** [comment]
Why is this branch not merged? I find it an useful update. I was following this tutorial, and the author has the same complaint: http://teropa.info/blog/2016/08/08/angular-2-hot-loading-with-ngrx-store-and-webpack.html

**Comment by tonybergeron** [comment]
I also am interested in if this branch will be merged sometime soon.

**Comment by xeor** [comment]
Any news here? `rc.2` is starting to get quite old and unusable as well. This project is still on `beta.8` and havent had a commit in 9 months..

**Comment by istobarton** [comment]
Any update on the status here? Would love to see this incorporated! @MikeRyan52

**Pull request closed without merging**